frontend/server/build.gradle

```diff
@@ -1,5 +1,6 @@
 dependencies {
     implementation "io.netty:netty-all:${netty_version}"
+    implementation group: "io.netty", name: "netty-transport-native-kqueue", version: "4.1.50.Final"
```

Comment:
`io.netty:netty-transport-native-epoll` ?

Reply:
netty-all - contains all the required packages hence removed native lib.
@maaquib - Thanks for pointing it out!